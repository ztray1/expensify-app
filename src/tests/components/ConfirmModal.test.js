import React from "react";
import {shallow} from "enzyme";
import ConfirmModal from "../../components/ConfirmModal";

let selectedOption,handleCancelOption,handleConfirmRemoveOption, wrapper;

beforeEach(()=>{
    selectedOption=jest.fn();
    handleCancelOption=jest.fn();
    handleConfirmRemoveOption=jest.fn();
    wrapper=shallow(
        <ConfirmModal
        selectedOption={selectedOption} 
        handleCancelOption={handleCancelOption}
        handleConfirmRemoveOption={handleConfirmRemoveOption}
        />
    );
});

test("should render confirm correctly",()=>{
    expect(wrapper).toMatchSnapshot();
});