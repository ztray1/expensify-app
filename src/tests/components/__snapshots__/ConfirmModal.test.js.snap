// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render confirm correctly 1`] = `
<Modal
  ariaHideApp={true}
  bodyOpenClassName="ReactModal__Body--open"
  className="modal"
  closeTimeoutMS={0}
  contentLabel="selected Option"
  isOpen={true}
  onRequestClose={[Function]}
  parentSelector={[Function]}
  portalClassName="ReactModalPortal"
  shouldCloseOnOverlayClick={true}
  shouldFocusAfterRender={true}
>
  <h3>
    Are you sure to remove?
  </h3>
  <button
    className="button button--modal"
    onClick={[Function]}
  >
    Confirm
  </button>
  <button
    className="button button--modal"
    onClick={[Function]}
  >
    cancel
  </button>
</Modal>
`;
